README.md

```diff
@@ -11,7 +11,10 @@
 
 The JavaScript server that runs Service Workers according to the [Winter Community Group specification](https://wintercg.org/).
 
-> Note: WinterJS is not officially endorsed by WinterCG, despite sharing "Winter" in their name. There are many [runtimes supporting WinterCG](https://runtime-keys.proposal.wintercg.org/), WinterJS being one among those
+> Note: WinterJS is not officially endorsed by WinterCG, despite sharing "Winter" in their name. There are many [runtimes supporting WinterCG](https://runtime-keys.proposal.wintercg.org/), WinterJS being one among those.
+
+⚠️ WinterJS is **early, pre-release software**. It is currently not fully compliant with the WinterCG spec and there are missing APIs. Also, the runtime itself is still a work in progress, so you may encounter stability and performance issues. Is it not recommended to use WinterJS in production yet.
+For more information, see the API Compatibility section below.
```

Comment:
This is too scary to read. Also, we should not put it on the front, but near the API implementation